### DISCUSSION TITLE
Continuous Errors with Blockscout Usage

### DISCUSSION THREAD

**Discussion by crypto-le** [blockscout/blockscout#9611] [General]
I encountered a problem due to a hard fork upgrade of our private chain. I can get the data using eth_getBlockByNumber.

Request:

`{
  "jsonrpc": "2.0",
  "method": "eth_getBlockByNumber",
  "params": [
    "0x281490",
    true
  ],
  "id": 1
}
`
Response

`{
  "jsonrpc": "2.0",
  "id": 1,
  "result": {
    "baseFeePerGas": "0x4a817c800",
    "difficulty": "0x0",
    ...
    "transactions": [
      {
        "blockHash": "0xda61be527aacfbfd3b3d89bd426737f55a86ef8bc18d94cb6f052166ba4c49be",
        "blockNumber": "0x281490",
        ...
        "hash": "0xca6e16e961b96cfad07538061a98b611b467b1c4f4787624ac73e09818d0589a",
        ...
      }
    ],
    ...
  }
}
`
But when using eth_getTransactionReceipt to query the receipt, I get null.

Request:

`{
  "jsonrpc": "2.0",
  "method": "eth_getTransactionReceipt",
  "params": [
    "0xca6e16e961b96cfad07538061a98b611b467b1c4f4787624ac73e09818d0589a"
  ],
  "id": 1
}
`
Response:
`{
  "jsonrpc": "2.0",
  "id": 1,
  "result": null
}
`
This issue causes the Blockscout program to continuously output errors.

I noticed there's a missing_block_count entry for the data 2626713 - 2626704. When I delete this data, it seems to regenerate automatically. How can I resolve this error, or how can I configure Blockscout to ignore this problematic block height?

![image](https://github.com/blockscout/blockscout/assets/29742985/e632544c-a61f-48d8-ba7a-a1c903a0dbfc)

**Comment by crypto-le** [comment]


**Reply by sl1depengwyn** [reply]
That is a good suggestion actually, waiting for the feedback how it gone

**Reply by crypto-le** [reply]
According to Gimlu's answer, this error can be avoided by using the BLOCK_RANGES parameter to filter out bad blocks.
BLOCK_RANGES="1..2626703,2626705..latest"

The console no longer refreshes the log of this error.
The wrong block task in the missing_block_count table does not exist anymore.
